plugins/module_utils/utils.py

```diff
@@ -9,7 +9,18 @@
 from ansible_collections.sensu.sensu_go.plugins.module_utils import errors
 
 
-def sync(state, client, path, payload, check_mode):
+def do_differ(current, desired):
```

Comment:
So by default (here) you use do_differ function that only verifies the 1st level of keys. For some specific modules (event) you implement an enhanced function that goes into depth to check all nested fields as well.

=> I suggest you use the enhanced function by default, not just for the event module

Reply:
I tried this and tests were all passing, but I wanted a second opinion on this :smile: . Will apply that change.